lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -243,7 +242,7 @@ def deep_freeze(record)
 RSpec.configure do |config|
   config.after(:example) do |example|
     if example.exception&.is_a?(FrozenError)
-      example.exception.message << TestProf::LetItBe::FROZEN_ERROR_HINT
+      example.exception.message << TestProf::LetItBe::FROZEN_ERROR_HINT unless example.exception.message.frozen?
```

Comment:
> unless you think it's a good idea to peek at the object's internals

Yeah, I think it's fine to patch something related to exceptions (because it shouldn't affect the happy path 🙂)